platform/linux-generic/odp_system_info.c

```diff
@@ -443,7 +443,7 @@ int odp_sys_huge_page_size_all(uint64_t size[], int num)
 	dir = opendir("/sys/kernel/mm/hugepages");
 	if (!dir) {
 		ODP_ERR("Failed to open huge page directory\n");
```

Comment:
ODP_ERR -> ODP_DBG then.

Reply:
Why? It's still an error, but just not critical error.